.github/workflows/docker.yml

```diff
@@ -21,4 +21,8 @@ jobs:
             DEEPSPEECH_REPO=https://github.com/${{ github.repository }} \
             DEEPSPEECH_SHA=${{ github.sha }}
       - run: |
-          docker build -t app:${{ matrix.template }} -f Dockerfile.${{ matrix.template }} .
+          mkdir /tmp/empty
+      - run: |
+          cd /tmp/empty; docker build -t app:${{ matrix.template }} -f ${{ github.workspace }}/Dockerfile.${{ matrix.template }} .
```

Comment:
This feels like a hack compared to simply using the build context in the Dockerfile instead of cloning from scratch.

Reply:
Yes, but using the context means sending the `.git` repo over to the docker daemon, and that's a few GBs.